src/jni_core/src/jvm_runtime.rs

```diff
@@ -88,6 +88,18 @@ impl JavaVmWrapper {
             }
         }
 
+        // move risingwave-source-cdc to the head of classpath, because we have some patched Debezium classes
```

Comment:
It's somehow hacky to do it in this way, because this depends on the behavior of the default class loader of specific jvm implementation. Not sure whether or not all class loaders always take the first seen class in the jar list.

Maybe we can try using the maven shade plugin to include all classes in the original jar and exclude the specific classes patched in this PR, and then we can explicitly exclude the some dbz artifacts in the pom file. The way to use shade plugin can be found in this [example](https://stackoverflow.com/questions/45683587/exclude-classes-from-being-included-in-shaded-jar)

Reply:
> It's somehow hacky to do it in this way, because this depends on the behavior of the default class loader of specific jvm implementation. Not sure whether or not all class loaders always take the first seen class in the jar list.

This concern can be eliminated by testing. Since we use same docker image in our ci pipeline and the cloud deployment, the behavior of jvm should be deterministic across our ci and cloud. 

> Maybe we can try using the maven shade plugin to include all classes in the original jar and exclude the specific classes patched in this PR, and then we can explicitly exclude the some dbz artifacts in the pom file. The way to use shade plugin can be found in this [example](https://stackoverflow.com/questions/45683587/exclude-classes-from-being-included-in-shaded-jar)

I think it is possible shadow original classes with the shade plugin, but we can do it in future pr if it is really needed.